.github/actions/e2e_generate_app/action.yml

```diff
@@ -4,13 +4,9 @@ inputs:
     description: Project key to specify project to generate ("retail-app-demo", "retail-app-ext" or "retail-app-no-ext")
     required: true
     type: string
-  cwd:
-    required: false
-    default: ${PWD}
```

Comment:
Removing this here as the pwa-sfra-hybrid-tests repo now has a different method for generating apps that do not use this action